docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -90,7 +91,7 @@ for instance the MediaBundle defines the method as::
 
         public function getObjectMetadata($object)
         {
-            $provider = $this->pool->getProvider($object->getProviderName());
+            $provider = $this->getConfigurationPool()->getContainer()->get($object->getProviderName());
```

Comment:
why this change? does the previous code not work?

Reply:
Yes, the following exception occurs:
`An exception has been thrown during the rendering of a template ("Notice: Undefined property: App\Admin\AuthUserAdmin::$pool").`

Reply:
In fact the code works good in BaseMediaAdmin, where it comes from, but not in my own admin. So  I guess I should not change code, but add note to explain how to get provider in general?

Reply:
Done